examples/tools/stack-scroll-mouse-wheel.md

```diff
@@ -1,6 +1,37 @@
 ---
-layout: needs-example
+layout: stack-tool
 title: Stack Scroll MouseWheel Tool
 toolName: StackScrollMouseWheel
 toolType: stack
 ---
+
+<h2 class="title is-2">How to set up the {{page.toolName}} tool:</h2>
+
+{% highlight javascript %}
+// Init cornerstone tools
+cornerstoneTools.init()
+
+// Enable any elements, and display images
+const imageIds = [`image_id_1`,`image_id_2`,]
```

Comment:
This example may be better served by a stack of images that are part of the same series. The `Exotic Animals` study is fun/cool, but it's more common to scroll through images in the same series:

![cbfEZk9ftJ](https://user-images.githubusercontent.com/5797588/56087309-5c391300-5e36-11e9-99f8-34de01a5feda.gif)

Reply:
Is there that it is allowed to use in that case?

Reply:
https://viewer.ohif.org/studylist

- Filter `modality` to `CT`
- Right click a table row
    - `Export`

Reply:
Already there. I got Bellona's sample. Thank you @dannyrb